Added initial cloud files (#7338)
influxdata/docs-v2: content/influxdb3/cloud/admin/tokens/_index.md

---
title: Manage tokens
seotitle: Manage tokens in {{< product-name >}}
description: >
  Manage tokens to authenticate and authorize access to actions, resources, and
  data in your {{< product-name >}} instance.
menu:
  influxdb3_cloud:
    parent: Administer InfluxDB
weight: 104
influxdb3/cloud/tags: [tokens, security]
related:
  - /influxdb3/cloud/get-started/setup/
  - /influxdb3/cloud/reference/cli/influxdb3/
---

{{% product-name %}} uses token-based authorization to authenticate and
authorize actions in your instance.

> [!Note]
> #### Manage InfluxDB 3 Cloud tokens with the influxdb3 CLI
>
> In {{% product-name %}}, you create and manage tokens with the
> [`influxdb3` CLI](/influxdb3/cloud/reference/cli/influxdb3/) or the InfluxDB
> HTTP API.

## Token types

> 
> {{% product-name %}} supports the following token types:
> 
> - **Admin tokens**: Grant full administrative access to all actions and
>   resources in the instance.
>   - The first admin token on an instance is the _operator_ token. It never
>     expires and is managed by InfluxData--InfluxData provides you an initial
>     admin token to administer your instance.
>   - **Named admin tokens** are additional admin tokens. They're long-lived by
>     default and expire only if you set an expiration.
> - **Database tokens**: Grant scoped read and/or write access to specific
>   databases. Use database tokens to authorize applications that write or query
>   data.

Reviewer: Carry over the token-type detail and the "Store secure tokens" callout from the Enterprise token docs (`admin/tokens/admin/create`), and align the operator-token wording with the merged OAuth setup flow — customers log in via OAuth and InfluxData manages the operator token, so it's no longer "provided to you to administer your instance."

```suggestion
- **Admin tokens**: Grant full administrative access to all actions and
  resources in the instance.
  - **Operator token**: The first admin token on an instance. InfluxData
    manages the operator token for your {{% product-name %}} instance.
    - Never expires
    - Cannot be edited or deleted
  - **Named admin tokens**: Additional admin tokens that you create and manage.
    - Can be created, edited, and deleted
    - Long-lived by default and expire only if you set an expiration
    - Cannot modify or remove the operator token
- **Database tokens**: Grant scoped read and write access to specific
  databases. Use database tokens to authorize applications that write or query
  data.
  - Grant `read`, `write`, or both to one or more databases
  - Cannot perform administrative actions

> [!Note]
> #### Store secure tokens in a secret store
>
> Token strings are returned _only_ when you create the token.
> Store tokens in a **secure secret store**.
> Anyone with access to an admin token has full control over your
> {{% product-name %}} instance.
> If you lose a token string, recreate the token.
```

## Create a token

Before you create tokens, [configure the `influxdb3` CLI](/influxdb3/cloud/get-started/setup/#configure-the-influxdb3-cli)

> to connect to your instance and authenticate with your admin token.

Reviewer: Make the prerequisites an explicit list and surface the admin-privileges requirement here. Also reference logging in (OAuth) instead of "authenticate with your admin token" to match the get-started setup flow.

```suggestion
Before you create tokens:

- [Configure the `influxdb3` CLI](/influxdb3/cloud/get-started/setup/#configure-the-influxdb3-cli)
  and [log in to your instance](/influxdb3/cloud/get-started/setup/#log-in-to-your-instance).
- Make sure you have admin privileges. Creating tokens requires admin access.
  If your user doesn't have admin access, ask an administrator to create a token
  for you.
```

Create a **named admin token**:

<!--pytest.mark.skip-->

```sh
influxdb3 create token --admin --name "support-2026"
```

To set an expiration, add `--expiry` with a duration (for example, `90d` or
`1y`):

<!--pytest.mark.skip-->

```sh
influxdb3 create token --admin --name "temp-90d" --expiry 90d
```

Create a **database token** with scoped read and write permissions:

<!--pytest.mark.skip-->

```sh { placeholders="DATABASE_NAME" }
influxdb3 create token \
  --permission "db:DATABASE_NAME:read,write" \
  --name "Read/write token for DATABASE_NAME"
```

The raw token string is shown only once--copy it immediately and store it

> securely.

Reviewer: Lead with the action.

```suggestion
Copy the raw token string immediately and store it securely.
It's shown only once.
```
